airbyte-cdk/python/airbyte_cdk/sources/file_based/remote_file.py

```diff
@@ -3,24 +3,29 @@
 #
 
 from datetime import datetime
+from enum import Enum
 from typing import Optional
 
 from pydantic import BaseModel
 
 
+class FileReadMode(Enum):
```

Comment:
should this enum be in `file_type_parser` since it's a property of the parser?

Reply:
Makes sense to me. Moved.